alacritty/src/renderer/rects.rs

```diff
@@ -274,7 +297,8 @@ impl RectRenderer {
             gl::BindBuffer(gl::ARRAY_BUFFER, 0);
         }
 
-        Ok(Self { vao, vbo, program, rect_vertices: Vec::new(), curl_vertices: Vec::new() })
+        let vertices = [Vec::new(), Vec::new(), Vec::new(), Vec::new()];
```

Comment:
```suggestion
        let vertices = Default::default();
```

This should work, right? Just a lot shorter. Could maybe even be inlined again.